test/new/db/anal/arm

```diff
@@ -822,7 +822,7 @@ NAME=arm jump table
 FILE=../bins/elf/analysis/callback.elf
 CMDS=<<EOF
 af @ sym.input_handler2
-CC. @ 0x000105b8~?(7
+CC. @ 0x000105b8~?\(7
```

Comment:
these are necessary because `(`, `)` and other chars are considered special chars in the new parser. the old parser should work well because of recent change.